pybaseball/datahelpers/postprocessing.py

```diff
@@ -0,0 +1,41 @@
+from typing import List
+
+import numpy as np
+import pandas as pd
+
+
+def coalesce_nulls(data: pd.DataFrame, value: any = np.nan) -> pd.DataFrame:
+    # fill missing values with NaN
+    data.replace(r'^\s*$', value, regex=True, inplace=True)
+    data.replace(r'^null$', value, regex=True, inplace=True)
+
+    return data
+
+
+def columns_except(data: pd.DataFrame, columns: List[str]) -> List[str]:
+    return list(np.setdiff1d(data.columns, columns))
+
+
+def convert_numeric(data: pd.DataFrame, numeric_columns: List[str]):
+    # data.loc[data[numeric_cols] == ''] = None
+    # data[numeric_cols] = data[numeric_cols].astype(float)
+    # Ideally we'd do it the pandas way ^, but it's barfing when some columns have no data
+
+    for col in numeric_columns:
```

Comment:
Could we insert here a quick check for whether the column contains decimals or percentages, and then use that to determine whether float or integer?

Reply:
I thought about that as well once you mentioned #59 but was worried about performance on larger data sets to scan all values and then attempt to determine if they were all ints or all floats. We could certainly add it if you'd like.

Reply:
We shouldn't need to check all of them, we should only need to check the first (nonnull?) instance to determine.
Re performance, I was mainly worried about a potential major regression, I'm not trying to extract every last bit here.

Reply:
Actually I think I have an idea that would work well here. Let me try some things and I'll update the PR.

Reply:
Okay, I added some new code that will just try to determine the type at parse time. I reran my performance metrics, and surprisingly this is even faster!

```
pybaseball> python -m timeit -n 100 -r 5 -s "from team_fielding_timeit import run_new_tf" "run_new_tf()"
100 loops, best of 5: 40.5 msec per loop
```

As well this would likely address the issue in #59 if we inject the try_parse function into the read_csv code, possibly using the converters parameter.

Reply:
wowwwwwwww

Reply:
wowwwwwwwwwww